android/maplibreui/src/main/java/com/stadiamaps/ferrostar/maplibreui/views/overlays/LandscapeNavigationOverlayView.kt

```diff
@@ -36,9 +43,10 @@ import kotlinx.coroutines.flow.asStateFlow
 fun LandscapeNavigationOverlayView(
     modifier: Modifier,
     camera: MutableState<MapViewCamera>,
-    navigationCamera: MapViewCamera = navigationMapViewCamera(),
+    navigationCamera: MapViewCamera,
```

Comment:
I believe that making this a non-optional parameter was a net negative for code quality for not much benefit. It's way too easy to forget to pass it with this many layers, and in fact it was not actually passed down previously. I haven't changed the `progressViewSize` parameter in this PR, but I also strongly suspect that the same is true of that as well and we should require it at compile itme.